packages/mgt-components/src/components/mgt-todo/mgt-todo.scss

```diff
@@ -186,6 +179,24 @@
         }
       }
     }
+
+    &.date {
+      color: $task-detail-color;
+    }
+
+    &::after {
+      border-bottom: none;
+    }
+
+    &.dark {
```

Comment:
If these are being pulled up from their current level of nesting, then shouldn't the old nested code for these be removed?